Assets/Scripts/SpawnPosition/SpawnerObject.cs

```diff
@@ -0,0 +1,3 @@
+﻿public class SpawnerObject : MonoSingleton<SpawnerObject>
```

Comment:
What is the need for the SpawnerObject? You can get spawn positions(children) without using this script

Reply:
I dont want to drop any object but only the one which has spawner children, just extra precaution.